assets/js/components/user-input/UserInputPreviewGroup.js

```diff
@@ -173,39 +175,54 @@ export default function UserInputPreviewGroup( {
 			} ) }
 		>
 			<div className="googlesitekit-user-input__preview-group-title">
-				<p>{ title }</p>
-				<Link
-					secondary
-					onClick={ handleOnEditClick }
-					ref={ editButtonRef }
-					disabled={
-						isScreenLoading ||
-						( !! currentlyEditingSlug && ! isEditing )
-					}
+				<LoadingWrapper loading={ loading } width="340px" height="21px">
```

Comment:
This is also a bit of a divergence from the IB. It's not a strictly necessary one - I must confess that I initially misread the IB and added these `LoadingWrapper` instances _within_ `UserInputPreviewGroup` rather than around it. However, on reflection I don't think it's significantly messier this way round - it boils down to there being three slightly different uses of `LoadingWrapper` in `UserInputPreviewGroup` rather than three more uniform instances in `UserInputPreview`.

The net result is a more granular loading state, with an individual preview block per textual element rather than a more monolithic preview state.

So it is stands, it's this:
![image](https://github.com/google/site-kit-wp/assets/18395600/8e26c39f-fee1-4961-ac63-5b0dde4243b0)

As opposed to this:
![image](https://github.com/google/site-kit-wp/assets/18395600/e64543df-3a2d-4fba-89fb-1b0dcb36edeb)

With this being the loaded state:
![image](https://github.com/google/site-kit-wp/assets/18395600/e384e729-784b-40c7-b765-2ee4023c1aad)

I think it's preferable like this, but would be interested in the reviewer's thoughts here. I can easily change it to follow the IB more closely.